js/web/test/test-runner.ts

```diff
@@ -243,8 +243,14 @@ export class ModelTestContext {
       this.initializing = true;
 
       const initStart = now();
-      const session = await initializeSession(
-          modelTest.modelUrl, modelTest.backend!, modelTest.ioBinding, profile, sessionOptions || {}, this.cache);
+      const session = modelTest.backend == 'webnn' ?
+          await initializeSession(
+              modelTest.modelUrl, testOptions?.webnnOptions || 'webnn', modelTest.ioBinding, profile,
+              testOptions?.sessionOptions || {} || {}, this.cache) :
+          await initializeSession(
```

Comment:
I confess ignorable here about what's going on, but it seems weird for `initializeSession` to be called twice, one right after the other. 🤨

Reply:
Thanks - that looks clearer now.